org-code-javabuilder/protocol/src/main/java/org/code/protocol/JavabuilderContext.java

```diff
@@ -0,0 +1,63 @@
+package org.code.protocol;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class JavabuilderContext {
```

Comment:
nit: mind adding a quick doc on how this is meant to be used?

Reply:
done!